### PR TITLE
Meeting docs

### PR DESCRIPTION
**Pull request by rcthomas**
Started a meeting document directory with minutes from first meeting

### CODE REVIEW

File: meetings/2021-08-13.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    * JEPs (https://github.com/jupyter/enhancement-proposals) may arise from security discussions (here)
+                    * Telemetry use mostly for audit/reconstruction but also anomaly detection?
+                    * Conflict between security/privacy
+                * Coordinated release process that doesn't interfere with rapid release?
+                * Consideration: Companies that may not normally interact with open source may engage more on security
+                    * Some folks may have solved these issues internally
+                    * Security engagement from that sector can be accelerated
+                * Conflict in some contexts is between features and security
+                    * Every minor thing can be deemed insecure in some way
+                    * Balancing that negotaion between UX and sec is time consuming
```
Comment (manics) [inline meetings/2021-08-13.md:56]:
```suggestion
                    * Balancing that negotiation between UX and sec is time consuming
```
Small typo

**Review by Carreau — APPROVED** [review]
Thanks for doing all of that. I'll try to attend some of the next meetings. 

A few thoughts maybe for next time. 

 - List people who have submitted actual CVE as recognition. 
 - Supply chain attack and credential management as part of the org. 
 - Raise Security specific funds that could be hold in a numfocus Security Specific Fund accounts.

**Comment by rcthomas** [comment]
@Carreau I've added your suggestions to the potential topics for the next meeting

**Comment by rcthomas** [comment]
@rpwagner you ok with committing these minutes from last time?

**Review by Zsailer — APPROVED** [review]

**Review by rpwagner — APPROVED** [review]
Looks good! I like the guidelines at the beginning.

**Comment by rpwagner** [comment]
@rcthomas you may need to give me a brief intro to the process for the minutes and agenda. Should I go ahead and merge it if I'm the last reviewer next time?

**Comment by rcthomas** [comment]
Mainly just converting the hackmd into a md and adding it, getting feedback and then merging it in is all I have planned at this point.  I don't think we need any specific person to do it but I think we all prefer to have at least one other person look things over.

**Pull request merged**